projects/packages/sync/src/modules/class-options.php

```diff
@@ -364,7 +364,10 @@ private function filter_theme_mods( &$value ) {
 	public function jetpack_sync_core_icon() {
 		$url = get_site_icon_url();
 
-		require_once JETPACK__PLUGIN_DIR . 'modules/site-icon/site-icon-functions.php';
+		if ( defined( 'JETPACK__PLUGIN_DIR' ) ) {
```

Comment:
This needs a little more work than a check on defined.  Looking at the site-icon-functions file it defines the function `jetpack_site_icon_url`

We need to rework the entire jetpack_sync_core_icon function so that we don't try calling `jetpack_site_icon_url` if we don't have the full jetpack installed.

Reply:
Fixed.